Code/GraphMol/SubstructLibrary/substructLibraryTest.cpp

```diff
@@ -0,0 +1,199 @@
+//
+//  Copyright (C) 2001-2017 Greg Landrum and Rational Discovery LLC
```

Comment:
I guess I'm probably guilt of this, but the copyright is definitely wrong.
`Copyright (C) 2017 Greg Landrum` is more probable.